db/schema.rb

```diff
@@ -123,6 +129,23 @@
     t.index ["wine_id"], name: "index_reviews_on_wine_id", using: :btree
   end
 
+  create_table "sodas", force: :cascade do |t|
+    t.integer  "user_id"
+    t.integer  "country_id"
+    t.string   "name"
+    t.string   "brand"
+    t.string   "origin_data"
+    t.boolean  "is_diet"
+    t.boolean  "is_caffeinated"
+    t.string   "color"
+    t.integer  "users_id"
```

Comment:
Where this is coming from? I wonder if t.references :users gives it. We shouldn't have this columns, instead we are missing columns: creator_id and last_editor_id. Try to roll back migration and change migration in sodas. I wrote comments in it. Replace t.references :users, column: :creator_id && t.references :users, column: :last_editor_id with t.references :creator && t.references :last_editor accordingly.  
Then we only will need to add some code to User model. In User model we would want to add: 

class User < ActiveRecord::Base
  has_many :created_sodas, :class_name => 'Soda', :foreign_key => 'creator_id'
  has_many :last_edited_sodas, :class_name => 'Soda', :foreign_key => 'last_editor_id'
end